wasmVirtualMachine.js

```diff
@@ -372,11 +375,56 @@ function build_module(byte_code) {
                 i++;
                 decode = Leb.decodeUint32(byte_code, i);
                 i = decode.nextIndex;
-                size = decode.value;
+                expected_end = i + decode.value;
+
+                // get array of table descriptions
+                decode = Leb.decodeUint32(byte_code, i);
+                i = decode.nextIndex;
+                num_els = decode.value;
+
+                // get table description
+                for (let j = 0; j < num_els; j++) {
+                    // table element type
+                    type = byte_code[i];
+                    i++;
+
+                    // // sanity check
+                    // if (type != -0x10) {
+                    //     console.log("issue with table type, can only be anyfunc");
+                    //     return -1;
+                    // }
+                    
+                    // get resizable limits, first the flag
+                    decode = Leb.decodeUint32(byte_code, i);
```

Comment:
The flag is not a uint_32. It's just one byte
https://webassembly.github.io/spec/binary/types.html#limits